helpers/constants.py

```diff
@@ -2,17 +2,27 @@
 product_name = "Test Product Setup_"
 product_name_uri = "test_product_setup_"
 product_version = "1" # do not change this value as it is hardcoded in the tests further.
-
+product_version_uri = "1"# do not change this value as it is hardcoded in the tests further.
 
 #Edit metaadata payload data:
 documentUsecase = "Migrate"
 urlFragment = "test_url_fragment"
-searchKeywords = "rhel8, rhel, installation"
+searchKeywords = "rhel8,rhel,installation"
 
 #Edit metadata payload for an assembly:
 assembly_documentusecase = "Troubleshoot"
 assembly_urlfragment = "test_assembly_url"
-assembly_searchkeywords = "rhel8, rhel, troubleshoot installation"
+assembly_searchkeywords = "rhel8,rhel,troubleshoot installation"
+
+#Module metadata:
+module_abstarct = "Abstarct for API test module"
```

Comment:
there is a typo here, and also can you change it for "Abstract test: This is the abstract text via the asciidoc."